### ISSUE TITLE
[Accessibility][Name, role and value- Recognizer Type (LUIS)]: Role is not defined for the placeholder text in text editor of adaptive dialog.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2134]
**User Experience** 
It will create confusion for low vision user as the placeholder text in 'Text editor' looks like normal text as well as it can be deleted. 

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Navigate to the "Bot name.main" available under filter edit field and activate it.
5. Navigate to 'Adaptive dialog' available at right side of the pane.
6. Navigate to 'recognizer type' and select 'LUIS' in it.
7. Navigate to the edit field below and observe the issue.

**Actual:** 
The placeholder of the edit field under "Recognizer Type" looks like normal text. As well as it is defined in div due to which if user edit the edit field, pre-defined text is not automatically gets erased.
As well as link mentioned in the edit field is accessible.

**Expected:**
It should looks like placeholder so that when user navigate and edit into the edit field, content gets cleared. In code it should have role as placeholder.

**MAS Impacted:** [MAS4.1.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment:**
![placeholder looks like normal text](https://user-images.githubusercontent.com/60913131/75560807-e51b6600-5a6b-11ea-9f2b-e4fc5bf390b8.jpg)
[Uploading MAS4.1.2_Role is not defined for the placeholder text.zip…]()

**Comment by beyackle** [comment]
This doesn't look repro-able in the current version. The box is now "Default result" with a sensible default of "dialog.result", which is not a placeholder.

**Comment by ashish315** [comment]
As per new UI we are not able to locate the pane. So, HCL is closing the issue.
